### PR TITLE
Hide advanced options in new feed form

### PR DESCRIPTION
**Pull request by marienfressinaud**
Can be reviewed commit by commit.

Changes proposed in this pull request:

- Move advanced options in a `<details>` tag
- Remove the legends (not perfect, but it looked weird with the `<summary>` tag) and also the i18n `sub.feed.auth.configuration` key (now useless)
- Sync the RTL files (I think most of the changes have been forgotten in previous PRs?)

By default:

Capture d’écran de 2021-03-13 15-26-26

When "advanced" is clicked:

Capture d’écran de 2021-03-13 15-26-32

How to test the feature manually:

1. Open the http://localhost:8080/i/?c=subscription&a=add page
2. Click on "Advanced"
3. Check the hidden inputs are displayed

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [x] unit tests written (optional if too hard) N/A
- [x] documentation updated N/A

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/master/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

File: p/themes/base-theme/template.css [hunk 1]
```diff
@@ -212,6 +212,15 @@ td.numeric {
 	display: block;
 }
 
+.form-advanced-title {
+	width: 200px;
+	padding: 15px 0;
```
Comment (Alkarex) [inline p/themes/base-theme/template.css]:
```suggestion
	padding: 15px 0;
	width: 200px;
```

Fix stylelint

File: p/themes/base-theme/template.rtl.css [hunk 2]
```diff
@@ -211,6 +212,15 @@ td.numeric {
 	display: block;
 }
 
+.form-advanced-title {
+	width: 200px;
+	padding: 15px 0;
```
Comment (Alkarex) [inline p/themes/base-theme/template.rtl.css]:
```suggestion
	padding: 15px 0;
	width: 200px;
```

Fix stylelint

File: app/views/subscription/add.phtml [hunk 3]
```diff
@@ -44,69 +44,73 @@
...
-		</div>
 
-		<legend><?= _t('sub.feed.advanced') ?></legend>
-		<div class="form-group">
-			<label class="group-name" for="path_entries"><?= _t('sub.feed.useragent') ?></label>
-			<div class="group-controls">
-				<div class="stick">
-					<input type="text" name="curl_params_useragent" id="curl_params_useragent" class="extend" value="" placeholder="<?= _t('gen.short.blank_to_disable') ?>" />
+			<div class="form-group">
+				<label class="group-name" for="path_entries"><?= _t('sub.feed.useragent') ?></label>
```
Comment (Alkarex) [inline app/views/subscription/add.phtml]:
```suggestion
				<label class="group-name" for="curl_params_useragent"><?= _t('sub.feed.useragent') ?></label>
```

Small error from previous PR

File: app/views/subscription/add.phtml [hunk 4]
```diff
@@ -44,69 +44,73 @@
...
-				<select class="number" name="proxy_type" id="proxy_type"><?php
-					foreach(['' => '', 3 => 'NONE', 0 => 'HTTP', 2 => 'HTTPS', 4 => 'SOCKS4', 6 => 'SOCKS4A', 5 => 'SOCKS5', 7 => 'SOCKS5H'] as $k => $v) {
-						echo '<option value="' . $k . '">' . $v . '</option>';
-					}
-				?>
-				</select>
-				<div class="stick">
-					<input type="text" name="curl_params" id="curl_params" class="extend" value="" placeholder="<?= _t('gen.short.blank_to_disable') ?>" />
+			<div class="form-group">
+				<label class="group-name" for="path_entries"><?= _t('sub.feed.proxy') ?></label>
```
Comment (Alkarex) [inline app/views/subscription/add.phtml]:
```suggestion
				<label class="group-name" for="proxy_type"><?= _t('sub.feed.proxy') ?></label>
```

Small error from previous PR

**Review by Alkarex — APPROVED** [review]

**Comment by Alkarex** [comment]
Ups, `sub.feed.auth.configuration` is used in the subscription page, so it should not have been removed

**Comment by marienfressinaud** [comment]
Uh… I thought I checked with a `git grep` :/

**Pull request merged**